### PR TITLE
Testing module

### PR DESCRIPTION
**Pull request by Nahuel-Salvatierra**
# Summary

- Added testing module for loading fixtures.
- Modified necessary data to create an order.
- Conducted the corresponding test.
- Corrected ESLint and coverage issues in the process.

# Detail

In this pull request, the focus was on enhancing the testing capabilities by introducing a new testing module dedicated to loading fixtures. Additionally, data modifications were implemented to ensure proper order creation. The related test for this functionality was successfully conducted, validating the changes made.

# Reference

This pull request is associated with Jira ticket [PDR-298](https://pablistem.atlassian.net/jira/software/projects/PDR/boards/2/backlog?selectedIssue=PDR-298).

### CODE REVIEW

File: src/common/fixtures/category.ts [hunk 1]
```diff
@@ -1,16 +1,15 @@
-import { Category } from '../../../src/module/category/application/domain/category.entity';
-export const category1: Category = {
```
Comment (Fludeo) [inline src/common/fixtures/category.ts:2]: Create a type based on the real Category, so every time Category changes it makes you change the fixture. 
You can use typescript Partials or Exclude.  Or you can use mapper functions.

File: src/common/fixtures/item.ts [hunk 2]
```diff
@@ -5,33 +5,27 @@ export const item1 = {
...
-  product: product1,
-  order: order1,
   subTotal: 0,
-  createdAt: undefined,
-  updatedAt: undefined,
+  order: order1.id,
+  product: product1.id,
 };
 
 export const item2 = {
```
Comment (Fludeo) [inline src/common/fixtures/item.ts]: same for items and all fixures entities

**Review by Fludeo — CHANGES_REQUESTED** [review]

File: src/common/fixtures/item.ts (hunk 2 above)
Comment (Nahuel-Salvatierra) [reply src/common/fixtures/item.ts]: Done

File: src/common/fixtures/category.ts (hunk 1 above)
Comment (Nahuel-Salvatierra) [reply src/common/fixtures/category.ts:2]: Done

**Review by Fludeo — APPROVED** [review]

**Pull request merged**